src/memray/_memray/hooks.cpp

```diff
@@ -279,7 +276,7 @@ dlopen(const char* filename, int flag) noexcept
     void* ret = hooks::dlopen(filename, flag);
     if (ret) {
         tracking_api::Tracker::invalidate_module_cache();
-        if (filename && 0 != strstr(filename, "/_greenlet.")) {
+        if (filename && nullptr != strstr(filename, "/_greenlet.")) {
```

Comment:
Is this change necessary? My understanding is that `strstr` returns a null pointer if there is no match.

Reply:
Yeah, that's what I am changing :)